### PR TITLE
Use DFGVariable nanosecond time in Dyn

### PR DESCRIPTION
**Pull request by Affie**
needs https://github.com/JuliaRobotics/IncrementalInference.jl/pull/823

### CODE REVIEW

**Comment by Affie** [comment]
This and Masters pass, so merging

File: test/testDynPoint2D.jl [hunk 1]
```diff
@@ -14,9 +14,9 @@ using Test
 global fg = initfg()
 
 # add two point locations
-global v0 = addVariable!(fg, :x0, DynPoint2(ut=0))
+global v0 = addVariable!(fg, :x0, DynPoint2(); nanosecondtime=0)
 
-global v1 = addVariable!(fg, :x1, DynPoint2(ut=1000_000))
+global v1 = addVariable!(fg, :x1, DynPoint2(ut=1000_000); nanosecondtime=10^9)
```
Comment (dehann) [inline test/testDynPoint2D.jl:19]: Hi @Affie , just checking here again and wondering if you don't want to drop the `DynPoint(ut=1000000)`?  You should perhaps go as far as the rest of the code and just use the type `DynPoint2` here.  IIF will internally create the object when necessary.

Comment (Affie) [reply test/testDynPoint2D.jl:19]: Hello, I left it in to trigger and test the deprecation on it. 
I'll open an issue on T vs T() if that is what you mean.

Comment (Affie) [reply test/testDynPoint2D.jl:19]: https://github.com/JuliaRobotics/DistributedFactorGraphs.jl/issues/577

**Pull request merged**